cmd/origin.go

```diff
@@ -44,6 +44,36 @@ var (
 		RunE:         serveOrigin,
 		SilenceUsage: true,
 	}
+
+	// Expose the token manipulation CLI
+	originTokenCmd = &cobra.Command{
+		Use:   "token",
+		Short: "Manage Pelican origin tokens",
+	}
+
+	originTokenCreateCmd = &cobra.Command{
+		Use:   "create",
+		Short: "Create a Pelican origin token",
+		Long: `Create a JSON web token (JWT) using the origin's signing keys:
+Usage: pelican origin token create [FLAGS] claims
+E.g. pelican origin token create --profile scitokens2 aud=my-audience scope="read:/storage" scope="write:/storage"
```

Comment:
I don't care for the arbitrary syntax here.  For well-known claims, provide flags:
```
--audience=foo --scope=read:/storage --scope=write:/storage
```

Then, for arbitrary overrides, add a separate claim:

```
--claim=foo=bar
```